### ISSUE TITLE
Sub zero doesn't detect subtitles in a soft-linked folder

### ISSUE THREAD

**Issue by Cooad** [pannal/Sub-Zero.bundle#571]
Hello.
tldr -> If there's a symlink in the path to the folder with media file and subitles, Subzero acts as if there are no subtitles. Plex pleys them normaly.

Full version:
My media is organized in a pretty standard way, and the subtitles are stored in the filesystem, along with media. An example of a path to a tv episode would be:
library/Doctor Who/Season 05/Doctor Who - S0501 - title.mp4.
And the corresponding subtitle:
library/Doctor Who/Season 05/Doctor Who - S0501 - title.en.mp4.

I also have a script, that runs once a week and checks the modification dates of the "Season XX" folders. If a folder hasn't been modified in 30 days, it gets archived. That means moving the folder to a different harddrive, and leaving a soft link behind. That way, all the files are awailable at the same path as they were.

It all works great until I need to modify a subtitle that has been archived already. The subtitles are detected by plex, but subzero states "no subtitle in storage".
I'm confident in my permission skills, so that's not the issue here - plex user has both read and write permissions to the subtitle files

**Comment by pannal** [comment]
> And the corresponding subtitle:
> library/Doctor Who/Season 05/Doctor Who - S0501 - title.en.mp4.
That's a typo right? It's `.srt`.

And those subs are downloaded by SZ?

**Comment by Cooad** [comment]
Sure, it was a typo. Or rather, a lazy copy-pasting :)
Yes, these subtitles have been downloaded by Subzero around April/May.
This happened with all TV shows that have been archived by my script.
However movies don't seem to be affected (they are archived almost the same way - the last directory is also a soft-link to a different harddrive).

**Comment by pannal** [comment]
OK, I'll investigate. And that happens for *all* moved episodes?

**Comment by Cooad** [comment]
Ok, I think I know what is going on, and that is my fault :)
In my initial investigation, I just checked the day and month of the subtitles on the disk. I didn't notice the year = 2017...
I believe the problem was with my initial archive run. It took a long time -> plex created duplicates I had to remove later -> SZ lost information about the subtitle...
Sorry to have wasted your time on something I should have investigated better... 
However, it is a shame that SZ doesn't let the user to modify subtitles it hadn't downloaded.

**Comment by Cooad** [comment]
I just did a little premature archivization, and the part id has not changed.
I consider this issue closed, as it turned out it was my fault all along :)
